pkg/provisioner/ironic/ironic.go

```diff
@@ -1234,7 +1234,7 @@ func (p *ironicProvisioner) ironicHasSameImage(ironicNode *nodes.Node) (sameImag
 
 func (p *ironicProvisioner) buildManualCleaningSteps() (cleanSteps []nodes.CleanStep, err error) {
 	// Build raid clean steps
-	if p.bmcAccess.RAIDInterface() != "" {
+	if p.bmcAccess.RAIDInterface() != "" && p.bmcAccess.RAIDInterface() != "no-raid" {
```

Comment:
TBH I don't think it's correct to ever assume that the default (empty string) means `no-raid`, since that's only true in the case of one driver (that doesn't return "" - see #319). I think a real fix would be to explicitly set the RAID interface in every BMC driver and only check for `no-raid` here.

Reply:
Sure I can go and change the bmcs with "" to `no-raid` and update this here =)

Reply:
Do we want to look for the string `no-raid` or do we want a bmcAccess method like `RAIDSupported()` that returns a bool?

Reply:
@dhellmann I would say it's fair to look at `no-raid` since this would say raid it's not supported, so having `no-raid` as default instead of `""` would be better

Reply:
Done